config/scheduledfeed.go

```diff
@@ -105,6 +111,33 @@ func (sConfig *ScheduledFeedConfig) GetScheduledFeeds() (map[string]feeds.Schedu
 	return scheduledFeeds, nil
 }
 
+func (sConfig *ScheduledFeedConfig) GetEventHandler() (*events.Handler, error) {
+	var err error
+	if sConfig.EventsConfig == nil {
+		sConfig.eventHandler = events.NewNullHandler()
+	} else if sConfig.eventHandler == nil {
+		sConfig.eventHandler, err = sConfig.EventsConfig.ToEventHandler()
+		if err != nil {
+			return nil, err
+		}
+	}
+	return sConfig.eventHandler, nil
+}
+
+func (ec EventsConfig) ToEventHandler() (*events.Handler, error) {
+	var sink events.Sink
+	switch ec.Sink {
+	case events.LOGGING_SINK_TYPE:
+		sink = events.NewLoggingEventSink(log.New())
+	default:
+		return nil, fmt.Errorf(
+			"unknown sink type '%v' provided to events configuration",
+			ec.Sink,
```

Comment:
Could be nice to provide the available sink types here in the error message?

Reply:
This would be easy to do now but would add an additional burden to adding sinks which could easily be missed. I think that adding documentation for the currently supported sinks/publishers/feeds etc is something that should be done for supporting user configuration. https://github.com/ossf/package-feeds/issues/97 is open to track this.

Reply:
Agreed

Reply:
Great! Until #97 is resolved it may be worth just adding a small snippet to the example config in the `README` as part of this PR, highlighting it optionally enabled (not blocking on this however).